containers/python-3-anaconda-postgres/test-project/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+cd $(dirname "$0")
+
+source test-utils.sh vscode
+
+# Remote - Containers does not auto-sync UID/GID for Docker Compose,
+# so make sure test project prvs match the non-root user in the container.
+fixTestProjectFolderPrivs
```

Comment:
qq: I don't see this used on other places. Curious why here it is needed.

Reply:
we added `fixTestProjectFolderPrivs` after finding it recommended in the [docker-compose template dirs](https://github.com/microsoft/vscode-dev-containers/blob/main/container-templates/docker-compose/test-project/test.sh) -- will follow-up offline

Reply:
removed